packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -286,7 +286,8 @@ export interface TerminalServiceExt {
     getEnvironmentVariableCollection(extensionIdentifier: string): theia.EnvironmentVariableCollection;
 }
 export interface OutputChannelRegistryExt {
-    createOutputChannel(name: string, pluginInfo: PluginInfo): theia.OutputChannel
+    createOutputChannel(name: string, pluginInfo: PluginInfo): theia.OutputChannel,
+    createOutputChannel(name: string, pluginInfo: PluginInfo, options?: { log: true }): theia.LogOutputChannel
```

Comment:
`options` is not optional here.

Reply:
This is an internal interface, so we can make it whatever we want ;-). But you're right, it probably shouldn't be.